packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -438,7 +443,7 @@ export class DataStores implements IDisposable {
 			snapshotTree: undefined,
 			isRootDataStore: false,
 			createProps: props,
-			groupId,
+			groupId: loadingGroupId,
```

Comment:
Also change name in IFluidDataStoreContextProps for consistent naming throughout.

Reply:
And anywhere else (e.g. ISnapshotTree, etc.)

Reply:
Internally we would like to keep groupid, for api we can expose loadinggroupid if that makes it more understandable to the user. We should strive for smaller names in summaries and snapshots as they take space too. So lets keep groupid to be user internally and server contracts.

Reply:
We do this in other places for other APIs and it's just confusing - if you are concerned about snapshot size, then you should deviate only at the snapshot layer and include detailed tsdoc that explains the name difference and the equivalence.  I disagree with this choice but wouldn't block it.

But IFluidDataStoreContextProps has no bearing on the snapshot and should be changed to match regardless.

Reply:
Yes, I will change here and in container runtime.

Reply:
Also fwiw - even for the snapshot probably prefer something like lgid (loading group ID) for better parallel with the real name, more concise, less likely name collision with other "groupID" in the repo.

Reply:
Made changes in the container runtime to use loadingGroupId internally and keeping groupid in server contracts. 
I really don't like the name lgid. It really does not tell me anything and feels confusing. Let's just have groupId on the server contracts.